build-tools/automation/azure-pipelines.yaml

```diff
@@ -51,14 +66,20 @@ stages:
   # Check - "Xamarin.Android (Mac Build)"
   - job: mac_build_create_installers
     displayName: Build
-    pool: $(HostedMacMojave)
+    pool: $(OnPremMacMojave)
```

Comment:
Is `$(OnPremMacMojave)` using VMs, or *real* hardware?  See also PR #3875, in which the use of Mojave VMs appears to cause `ANDZA0000` errors while building.

(We still don't understand *why*.)

Reply:
OnPrem is physical hardware. I hope they're not affected by this D: